modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/CheckpointCommand.java

```diff
@@ -81,4 +109,28 @@ public class CheckpointCommand extends AbstractCommand<Void> {
     @Override public Void arg() {
         return null;
     }
+
+    /** {@inheritDoc} */
+    @Override public void parseArguments(CommandArgIterator argIter) {
+        nodeId = null;
+
+        while (argIter.hasNextSubArg()) {
+            String nextArg = argIter.nextArg("Unexpected error on parsing checkpoint command args");
+
+            if (nextArg.equals(NODE_ID_ARG_NAME))
+                nodeId = UUID.fromString(argIter.nextArg("failed to read node ID"));
+        }
+    }
+
+    /**  */
+    private String collectNodesInfo(GridClientCompute compute) throws GridClientException {
+        SB b = new SB();
+
+        b.a("Node with id=" + nodeId + " not found in the topology").nl()
+            .a("Available cluster nodes:").nl();
+
+        compute.nodes((n) -> n.connectable() && !n.isClient()).forEach(node1 -> b.a(node1.nodeId()).nl());
```

Comment:
In first lambda you use `n`, in second you use `node1`. What's the reason of this discrepancy? Please use the same name, it would be easier to read

Reply:
fixed